src/scg.c

```diff
@@ -2119,6 +2119,8 @@ int igraph_scg_laplacian(const igraph_t *graph,
     igraph_bool_t tmp_lsparse = !Lsparse, tmp_rsparse = !Rsparse;
     igraph_sparsemat_t myLsparse, myRsparse, tmpsparse, Rsparse_t;
 
+    IGRAPH_UNUSED(direction);
```

Comment:
We should create a wishlist issue to implement the function of this parameter, otherwise it will get forgotten. IMO, this should be done for every similar case (non-functional documented arguments in public functions)

Reply:
Done: https://github.com/igraph/igraph/issues/1541